settings/settingsfakes/fake_platform_settings_getter.go

```diff
@@ -128,7 +128,7 @@ func (fake *FakePlatformSettingsGetter) GetAgentSettingsPathReturnsOnCall(i int,
 	}{result1}
 }
 
-func (fake *FakePlatformSettingsGetter) GetDefaultNetwork() (settings.Network, error) {
+func (fake *FakePlatformSettingsGetter) GetDefaultNetwork(isIpV6 bool) (settings.Network, error) {
```

Comment:
can we choose `ipv6`, `isIpv6` (or perhaps `isIPv6`)... 

`is_ipv6` seems wrong in that Golang suggest camelCase for variables, as does `IsVersion6` (see `func (ia failingInterfaceAddress) GetIP(IsVersion6 bool) (string, error)`)